src/config/env.config.js

```diff
@@ -12,6 +12,10 @@ const env = {
   adminEmail: process.env.ADMIN_EMAIL || 'admin@hitmoments.com',
   jwtSecret: process.env.JWT_SECRET || 'hitmoments.com',
   jwtExpire: process.env.JWT_EXPIRE || '1d',
+  apiUrl: process.env.API_URL || `http://localhost:${process.env.PORT || 3000}`,
+  frontendUrl: process.env.FRONTEND_URL,
+  jwtEmailSecret: process.env.JWT_EMAIL_SECRET,
+  jwtEmailExpire: process.env.JWT_EMAIL_EXPIRE || '1h',
```

Comment:
ứng dụng mình có thể sẽ có thêm một số loại gửi email cần đến thời gian sống, khóa nên em, đặt rõ tên ra nữa
```suggestion
  jwtEmailSecret: process.env.JWT_VERIFY_EMAIL_SECRET,
  jwtEmailExpire: process.env.JWT_VERIFY_EMAIL_EXPIRE || '1h',
```